package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this supposed to be here? Appear to have two `package-lock.json` files now?

Reply:
No, I must have run npm commands in this directory by mistake, I've deleted it.